clientd/src/rpc/responses.rs

```diff
@@ -3,7 +3,13 @@ use fedimint_client::mint::SpendableNote;
 use serde::Serialize;
 use std::collections::BTreeMap;
 
-#[derive(Serialize)]
+#[derive(Debug, Serialize)]
+pub enum HandlerResult {
```

Comment:
Should probably be called `HandlerResponse` to

1. Be in line with `InfoResponse`
2. Not be confused with `std::result::Result`